bundles/paikkatietoikkuna/admin/admin-layeranalytics/Flyout.js

```diff
@@ -35,19 +37,52 @@ Oskari.clazz.define('Oskari.framework.bundle.admin-layeranalytics.Flyout',
 
             this.updateListing();
         },
+        generateToScaleURL (details) {
+            const stack = details.stack[0];
+            let toScaleURL = '/?coord=' + stack.x + '_' + stack.y;
+            toScaleURL += '&mapLayers=';
+
+            for (const [index, value] of stack.layers.entries()) {
+                toScaleURL += value; // add layer id
+                toScaleURL += '+100'; // add layer opacity
+                toScaleURL += '+'; // add layer default style as empty string
+                if (index !== (stack.layers.length - 1)) {
+                    toScaleURL += ','; // add layer separator if not last layer in stack
+                }
+            }
+
+            return toScaleURL;
+        },
```

Comment:
Is there a reason we send this as reference to the component instead of it being part of the component? It only uses the first entry on stack which is problematic so if we want to keep it here we should just send the stack not the whole details. 

Also:
```suggestion
        generateToScaleURL (details) {
            const stack = details.stack[0];
            let toScaleURL = '/?coord=' + stack.x + '_' + stack.y;
            // [layer id]+[opacity as 100]+[style as empty string]
            return toScaleURL + '&mapLayers=' + stack.layers.map(id => id + `+100+`).join(',');
        },
```

Reply:
Moved.